helix-term/src/commands.rs

```diff
@@ -2928,7 +2978,21 @@ pub mod cmd {
             doc: "Go to line number.",
             fun: goto_line_number,
             completer: None,
-        }
+        },
+        TypableCommand {
+            name: "sort",
+            aliases: &[],
+            doc: "Sort lines in selection.",
+            fun: sort,
```

Comment:
I'm not sure how much appetite @archseer has for doing GNU style command arguments, but if so, this could just be a single command that takes an `-r` flag to reverse?

Reply:
I don't have a set opinion here, I did consider it might be nice to have flag parsing for typed commands in the future.